apps/condo/domains/ticket/components/BaseTicketForm/index.tsx

```diff
@@ -184,12 +184,9 @@ export const BaseTicketForm: React.FC<ITicketFormProps> = (props) => {
                                                                 property={property}
                                                                 loading={loading}
                                                                 allowClear={true}
-                                                                onClear={() => {
-                                                                    form.setFieldsValue({ sectionName: null, floorName: null })
-                                                                }}
-                                                                onSelect={(_, option) => {
-                                                                    setSelectedUnitName(option.key)
-                                                                    updateSectionAndFloor(form, option.key)
+                                                                onChange={(_, option) => {
+                                                                    setSelectedUnitName(option?.key ?? null)
```

Comment:
Looks confusing for me. Do you really need such checks?
```
option?.key ?? null
```

Reply:
@dkoviazin I want to be sure that value is exactly null, not undefined.

Reply:
Try to write self documented code. If it takes 2 more lines of code and everybody can understand from first look what's going on here. 
Maybe this will help you to understand why self documented code is better then strange symbols sequence :)

https://www.linux.org.ru/forum/development/392747

Reply:
@dkoviazin changed to more obvious construction